apps/apps.humanatlas.io/src/app/pages/us1/cell-population-predictor/cell-population-predictor.component.html

```diff
@@ -0,0 +1,53 @@
+<div class="container">
```

Comment:
You don't need a container div. The component itself is the container so place container styles directly under the `:host` selector in scss, i.e.
```scss
:host {
  /* Container styles goes here */
}
```